### PR TITLE
Changes for name & description of device, add info in yosensi index.yaml

### PR DESCRIPTION
**Pull request by yosensi**
#### Summary

...

#### Changes

- added new photo for YO Agri Box
- changed name & description of each device
- added new hardware version in YO Pure Pro
- added info about yosensi in index.yaml
- changed photo for YO Pulse device 

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
Hi @yosensi please resolve the branch conflict, so i can review all. thanks

File: vendor/yosensi/yo-360.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+name: YO 360 - temperature, humidity monitoring & accelelometr | LoRa EU868, US915, AU915, AS923
+
+description: YO 360 measures temperature and humidity, and it sends its own position on the x, y, and z axes (via a built-in accelerometer). Its small size, battery power supply, and wide data transmission range allow the device to be used virtually anywhere. Additionally, it comes in two enclosure types one for indoor use (material ABS) and another for outdoor use (epoxy resin). It is equipped with a lithium battery with a very low level of self-discharge. The device provides low power consumption and longer battery life. YO 360 is a operating over LoRaWAN and is high-quality product made in EU.
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-360.yaml:3]: Can you write a shorter description?

File: vendor/yosensi/yo-agribox.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+name: YO AgriBox - remote measurement of soil moisture at 3 points | LoRa EU868, US915, AU915, AND AS923
+description: YO AgriBox is a LoRaWAN device for measuring soil moisture at 3 points. Additionally, the device contains sensors that measure temperature and relative humidity inside the device. YO AgriBox makes it easy to plan and maintain optimum growing conditions for plants.
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-agribox.yaml:2]: 'description` should be only one line (line 3, 4,...), can you make it shorter?

File: vendor/yosensi/yo-agribox.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+# Product and data sheet URLs (optional)
+productURL: https://yosensi.io/products/yo_agri_box/
+dataSheetURL: https://yosensi.io/support/datasheets/YO_AgriBox_datasheet.pdf
+
+
+
+# Photos
+photos:
+  main: YO_AgriBox_01C.png
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-agribox.yaml:87]: please review the name of the file, also should be all in lowercase `yo-agribox-01c` also rename the file to lowercase

File: vendor/yosensi/yo-analog.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
+name: YO Analog - analog signal monitoring, voltage input (0–10 V) or current input (4–20 MA) | LoRa EU868, US915, AU915, AND AS923
+description: YO Analog makes it possible to monitor devices and processes in automation. The device has six configurable measurement inputs, each of which can operate in one of two analogue modes voltage input (0–10 V) or current input (4–20 mA).
```
Comment (Jaime-Trinidad) [inline vendor/yosensi/yo-analog.yaml:2]: try to reduce description, and only should be one line

Comment (yosensi) [reply vendor/yosensi/yo-analog.yaml:2]: What is character limit for the description?

Comment (Jaime-Trinidad) [reply vendor/yosensi/yo-analog.yaml:2]: There is no limit, but for example in the name of the device it is mention `voltage input` and `current input`, also in description, you can take it out from one of those

**Comment by yosensi** [comment]
I did some changes in next commit can you check it and aprove it?

**Comment by Jaime-Trinidad** [comment]
hey @yosensi there are still conflicts in this PR, can you resolve them?

**Comment by yosensi** [comment]
I deleted and added new photos, and I think I made comit to reslove conflicts

**Review by yosensi — COMMENTED** [review]
changed format of pictures deleted old one, in profile.yaml for every region and device all data are correct

**Pull request closed without merging**